i18n/config.ts

```diff
@@ -64,20 +63,7 @@
     // in case window.VEDACCIO_LANGUAGE is undefined,it will fall back to 'en-US'
     lng: window?.__VERDACCIO_BASENAME_UI_OPTIONS?.language,
     fallbackLng: 'en-US',
-    whitelist: [
-      'en-US',
-      'cs-CZ',
-      'pt-BR',
-      'es-ES',
-      'de-DE',
-      'fr-FR',
-      'zh-CN',
-      'ja-JP',
-      'ru-RU',
-      'uk-UA',
-      'km-KH',
-      'zh-TW',
-    ],
+    whitelist: ['en-US', 'cs-CZ', 'pt-BR', 'es-ES', 'de-DE', 'fr-FR', 'zh-CN', 'ja-JP', 'ru-RU', 'uk-UA', 'km-KH', 'zh-TW'],
```

Comment:
This seems a Prettier problem, is working properly?

Reply:
my bad :) don't use `--no-verify` 😋